From 3605eb072cfbf604785e4918dcb50b78dd016c62 Mon Sep 17 00:00:00 2001
From: Dyneteve <dyneteve@pixelexperience.org>
Date: Fri, 24 Jul 2020 23:09:51 +0430
Subject: [PATCH] gralloc: Fix RAW for trinket

---
 gralloc/gr_utils.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gralloc/gr_utils.cpp b/gralloc/gr_utils.cpp
index 4306663ee..78240141c 100644
--- a/gralloc/gr_utils.cpp
+++ b/gralloc/gr_utils.cpp
@@ -1020,10 +1020,10 @@ void GetAlignedWidthAndHeight(const BufferInfo &info, unsigned int *alignedw,
       aligned_w = ALIGN(width, 16);
       break;
     case HAL_PIXEL_FORMAT_RAW12:
-      aligned_w = ALIGN(width * 12 / 8, 16);
+      aligned_w = ALIGN(width * 12 / 8, 8);
       break;
     case HAL_PIXEL_FORMAT_RAW10:
-      aligned_w = ALIGN(width * 10 / 8, 16);
+      aligned_w = ALIGN(width * 10 / 8, 8);
       break;
     case HAL_PIXEL_FORMAT_RAW8:
       aligned_w = ALIGN(width, 16);
